roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -13,108 +13,64 @@
   loop: "{{kafka_broker_packages}}"
   when: ansible_os_family == "Debian"
 
-# Configure environment
-- name: broker group
+# Configure Environment
+- name: Kafka Broker group
   group:
     name: "{{kafka_broker.group}}"
-- name: broker user
+
+- name: Kafka Broker user
   user:
     name: "{{kafka_broker.user}}"
     comment: "Kafka User"
     system: yes
     group: "{{kafka_broker.group}}"
-- name: Change keytabs ownership
+
+- name: Create Keytabs Directory
   file:
     path: "{{kerberos.keytab_dir}}"
     group: "{{kafka_broker.group}}"
     mode: '650'
     recurse: yes
-- name: create broker data directories
+  when: sasl_protocol == 'kerberos'
+
+- name: Copy in Keytab File
+  copy:
+    src: "{{kafka_broker_kerberos_keytab_path}}"
+    dest: "{{kerberos.keytab_dir}}/{{kafka_broker_kerberos_keytab_path | basename}}"
+    mode: '600'
+    owner: "{{kafka_broker.user}}"
+    group: "{{kafka_broker.group}}"
+  when: sasl_protocol == 'kerberos'
+  notify:
+    - restart kafka
+
+- name: Create Kafka Broker Data Directories
   file:
     path: "{{item}}"
     owner: "{{kafka_broker.user}}"
     group: "{{kafka_broker.group}}"
     state: directory
     mode: 0755
   with_items: "{{kafka_broker.datadir}}"
-- name: broker plaintext config
+
+- name: Create Kafka Broker Config
   template:
     src: server.properties.j2
     dest: "{{kafka_broker.config_file}}"
     mode: 0640
     owner: "{{kafka_broker.user}}"
     group: "{{kafka_broker.group}}"
-  when: security_mode == "plaintext"
-  notify:
-    - restart kafka
-- name: broker ssl config
-  template:
-    src: server_ssl.properties.j2
-    dest: "{{kafka_broker.config_file}}"
-    mode: 0640
-    owner: "{{kafka_broker.user}}"
-    group: "{{kafka_broker.group}}"
-  when: security_mode == "ssl"
-  notify:
-    - restart kafka
-- name: broker sasl_ssl config
-  template:
-    src: server_sasl_ssl.properties.j2
-    dest: "{{kafka_broker.config_file}}"
-    mode: 0640
-    owner: "{{kafka_broker.user}}"
-    group: "{{kafka_broker.group}}"
-  when: security_mode == "sasl_ssl"
-  notify:
-    - restart kafka
-- name: broker ssl custom signed config
-  template:
-    src: server_ssl_self_signed.properties.j2
-    dest: "{{kafka_broker.config_file}}"
-    mode: 0640
-    owner: "{{kafka_broker.user}}"
```

Comment:
Why are refactoring how all of the templates work?  This PR should just be updating Kerberos to use better encryption types, and make the krb5.conf configurable.